init_conditions/subcell_agent_generator.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import math
+import argparse
+import imageio
+
+import numpy as np
+import pandas as pd
+import quilt3
+from matplotlib import pyplot as plt
+from aicsimageio import AICSImage
+from hexalattice.hexalattice import create_hex_grid
+
+
+OUTPUT_COLUMNS = ["x", "y", "z", "id"]
+SCALE_MICRONS = 0.108333  # from metadata.csv
+
+
+class SubcellAgentGenerator:
+    """
+    Download and read AICS segmented cell fields and
+    generate 3D positions and identities of initial subcell agents
+    on a cartesian or hexagonal grid
+    """
+    @staticmethod
+    def download_fov_images(output_path="", max_images=10):
```

Comment:
This function looks great! Love the use of quilt instead of lkaccess and the random selection (with nice handling, like skipping existing FOVs etc). Awesome.